crowns.lic

```diff
@@ -122,22 +122,30 @@ class Dice
       if in_hand
         case in_hand
         when *@trash_items
-          DRC.bput("put my #{in_hand} in bucket", 'You put', 'You drop')
+          DRC.bput("put my #{in_hand} in bucket", 'You put', 'You drop', 'What were', 'Stow what?')
         else
           # Coil rope so we can store it
           if /\brope\b/ =~ in_hand
             fput("coil my #{in_hand}")
           end
-          case DRC.bput("stow #{in_hand} in my #{@hollow_eve_loot_container}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-          when 'to fit in the'
-            DRC.message("*** Item is too big to fit in your stow container! ***")
-            exit
+          DRCI.put_away_item?(in_hand, @hollow_eve_loot_container)
+          if reget(5, /to fit in the/)
```

Comment:
💡 The `?` in `put_away_item?` means that it returns a boolean (true/false). You could capture that and use it instead of `reget`. However, that would mean you couldn't differentiate these two cases and instead have a more generic 'couldn't put away' error message